public/js/hotkey_binder/model/addShortcut.js

```diff
@@ -0,0 +1,86 @@
+//Assign the callback func for the keymap here
+
+const addShortcut = (keys, action) => {
+	let callback;
+	switch (action) {
+		case "New Project":
+			callback = newProject;
+			break;
+		case "Save Online":
```

Comment:
Maybe have a JSON map instead of using switch case? 

like 
```
shortcutMap = {
    "New Project": newProject
};
// Then u can simply do 
if(action in shortcutMap) {
    return shortcutMap[action];
}
```

Reply:
I'll try on implementing as suggested.